_posts/en/pages/2016-01-18-segwit-wallet-dev.md

```diff
@@ -0,0 +1,124 @@
+---
+version: 1
+title: Segregated Witness Wallet Development Guide
+name: segwit-wallet-dev
+type: page
+permalink: /en/segwit_wallet_dev
+---
+
+### Witness program
```

Comment:
"Witness program" should not be the first section. An intro section would be nice that links to something else which explains segwit, maybe @sipa's presentation or whatever, like:

http://diyhpl.us/wiki/transcripts/scalingbitcoin/hong-kong/segregated-witness-and-its-impact-on-scalability/

... if there is a page on bitcoin.org then we should prefer linking to that, instead of the transcript.